.gitignore

```diff
@@ -0,0 +1,3 @@
+.gradle/
```

Comment:
What is the point to add additional configuration? I like current gitignore setup because it is minimal, simple and cover all untracked files.